README.md

```diff
@@ -5,6 +5,35 @@
 
 # DiracX-Web
 
+## Overview
+
+`diracx-web` is a web application that provides a user interface to interact with the [DiracX](https://github.com/DIRACGrid/diracx) service and is part of the [DIRAC](https://github.com/DIRACGrid) ecosystem.
+
+This repository is organized as a monorepo, with the following key packages:
+
+- [diracx-web](./packages/diracx-web)
+- [diracx-web-components](./packages/diracx-web-components)
+- [diracx-web-extension-example](./packages/diracx-web-extension-example)
+
+![Architecture diagram](./docs/architecture_overview.png)
+
+### Packages
+
+- **DiracX-Web**: A Next.js application that serves as a demo to showcase the default functionalities of the `DiracX-Web-Components` package.
+
+- **DiracX-Web-Components**: A library of reusable React components designed for integration within the `DiracX-Web` package and to facilitate the creation of custom DiracX web extensions.
+
+- **DiracX-Web-Extension-Example**: An illustrative example of a web extension demonstrating how to extend the functionality of `DiracX-Web` using the components from the `DiracX-Web-Components` package.
+
+### Project Management
+
+- **Lerna**: Utilized for efficient monorepo management.
```

Comment:
Can you briefly describe why it is needed in our context?

Reply:
Also I think you can add a reference to the documentation of each tool (lerna, npm, release-please) such as:

```md
- [**Lerna**](<url lerna documentation>): Utilized ...
```